props/common.props

```diff
@@ -1,5 +1,9 @@
 <Project>
   <PropertyGroup>
+    <HasCommonProperties>true</HasCommonProperties>
```

Comment:
Is this really necessary?

I'm not sure whether you want to eventually merge this (your OP states that you don't), but either way, can't we just make sure that project files include all the properties they need? This really shouldn't be an issue at all, ever... Plus I can't imagine tons of projects being created..